### ISSUE TITLE
Bug:  HTTP2 doesn't work

### ISSUE THREAD

**Issue by atimin** [reductstore/reduct-rs#4]
### What happened?

The SDK uses reqwest, which uses HTTP2 automatically for HTTPS. It doesn't work properly in the SDK and has panicked tokio threads:

```
thread 'tokio-runtime-worker' panicked at /home/atimin/.cargo/registry/src/index.crates.io-6f17d22bba15001f/reduct-rs-1.9.2/src/record/query.rs:207:55:
called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Body, source: hyper::Error(Body, Error { kind: Io(Kind(ConnectionReset)) }) }
```
 

### Version

1.9.2

### OS

Linux

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by atimin** [comment]
The current workaround is to disable HTTP2 with the `only_http1()` option.

**Comment by atimin** [comment]
Solved after the `rewest` update in  #8
